crates/rpc-types/src/eth/log.rs

```diff
@@ -14,6 +14,9 @@ pub struct Log<T = LogData> {
     /// Number of the block the transaction that emitted this log was mined in
     #[serde(with = "alloy_serde::u64_hex_opt")]
     pub block_number: Option<u64>,
+    /// The timestamp of the block.
+    #[serde(skip_serializing_if = "Option::is_none", with = "alloy_serde::u64_hex_opt", default)]
+    pub block_timestamp: Option<u64>,
```

Comment:
can we reference the EIP for this here?

Reply:
i dont think it's yet an EIP, so should i just link to [the proposal](https://ethereum-magicians.org/t/proposal-for-adding-blocktimestamp-to-logs-object-returned-by-eth-getlogs-and-related-requests)?

Reply:
right then let's add this and https://github.com/ethereum/execution-apis/issues/295

Reply:
updated